src/pages/About.jsx

```diff
@@ -1,152 +1,145 @@
+                      href={socials[0].url}
+                      target="_blank"
+                      rel="noreferrer"
+                    >
+                      <FaGithubSquare />
+                    </a>
+                    <a
+                      className="transition ease-in-out duration-300 origin-bottom hover:text-lava-900 hover:scale-110"
+                      href={socials[1].url}
+                      target="_blank"
+                      rel="noreferrer"
+                    >
+                      <FaLinkedin />
+                    </a>
+                  </article>
                 </div>
+                <p className="z-10">{summary}</p>
+              </div>
 
-                {index !== 0 && (
-                  <>
-                    {index % 2 !== 0 ? (
-                      <>
-                        <SvgComponent className="absolute top-0 left-0 rotate-180 -scale-x-100 md:hidden" />
-                        <SvgComponent className="absolute -top-8 left-0 -scale-x-100 md:hidden" />
-                      </>
-                    ) : (
-                      <>
-                        <SvgComponent className="absolute top-0 left-0 rotate-180 md:hidden" />
-                        <SvgComponent className="absolute -top-8 left-0 md:hidden" />
-                      </>
-                    )}
-                  </>
-                )}
-              </article>
-            ))}
-          </div>
+              {/* {index !== 0 && (
+                <>
+                  {index % 2 !== 0 ? (
+                    <>
+                      <SvgComponent className="absolute top-0 left-0 rotate-180 -scale-x-100 md:hidden" />
+                      <SvgComponent className="absolute -top-8 left-0 -scale-x-100 md:hidden" />
+                    </>
+                  ) : (
+                    <>
+                      <SvgComponent className="absolute top-0 left-0 rotate-180 md:hidden" />
+                      <SvgComponent className="absolute -top-8 left-0 md:hidden" />
+                    </>
+                  )}
+                </>
+              )} */}
+              {index !== team.length - 1 && (
+                <SvgComponent className="absolute -bottom-1 left-0 md:hidden" />
+              )}
+              {index !== 0 && (
+                <SvgComponent className="absolute -top-1 left-0 rotate-180 md:hidden" />
+              )}
+            </article>
+          ))}
+        </div>
 
-          <SvgComponent className="absolute -top-8 left-0" />
-          <SvgComponent className="absolute top-0 left-0 rotate-180" />
-        </section>
-      </div>
+        {/* <SvgComponent className="absolute -top-8 left-0" />
+          <SvgComponent className="absolute top-0 left-0 rotate-180" /> */}
+        <SvgComponent className="absolute -top-1 left-0 rotate-180" />
+      </section>
+      {/* END Team section */}
```

Comment:
Svg components doesn't have a color associated with them. You can add `fill-{color}` to color them. I believe the color of the waves on the homepage is #FFC945.

Reply:
Thanks, Tim. I added the class `fill-[#FFC945]` to each of the wave components.